crates/sovereign-sdk/full-node/sov-stf-runner/src/prover_service/mod.rs

```diff
@@ -93,6 +93,7 @@ pub trait ProverService<Vm: Zkvm> {
     async fn submit_witness(
         &self,
         input: Vec<u8>,
+        assumption: Option<Vec<u8>>,
```

Comment:
rename submit_witness to submit_input

add new method called submit_assumption

Reply:
how about submit_input_and_assumptions?
Because current implementation depends on receiving the witness to start proving